railties/lib/rails/generators/rails/plugin/templates/lib/tasks/%namespaced_name%_tasks.rake

```diff
@@ -1,4 +1,4 @@
 # desc "Explaining what the task does"
-# task :<%= name %> do
+# task :<%= underscored_name %> do
```

Comment:
Not sure what's the "right" thing for this, perhaps it should be rake-`namespace`'d too? (However, this is commented out, so perhaps it's not a big deal.